lmdeploy/cli/utils.py

```diff
@@ -261,7 +261,7 @@ def engine(parser):
 
         return parser.add_argument('--engine',
                                    type=str,
-                                   default='turbomind',
+                                   default=None,
```

Comment:
this change will affect the behavers of all cli. You should add an argument of `default` in line 259 and pass `default=None` when calling it in `add_parser_list`